vnext/src/Libraries/Components/TextInput/WindowsTextInputNativeCommands.windows.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Consider including codegenNativeComponent in this file and naming it *NativeComponent.js. That has been our convention, not just exporting commands and naming it NativeCommands

Reply:
Sure I can rename the file. About codegenNativeComponent - looking at AndroidTextInputNativeComponent, RCTMultilineTextInputNativeComponent.js etc. I saw they only imported  codegenNativeCommands.

```
export const Commands: NativeCommands = codegenNativeCommands<NativeCommands>({
  supportedCommands: ['focus', 'blur', 'setTextAndSelection'],
});

const SinglelineTextInputNativeComponent: HostComponent<mixed> = requireNativeComponent<mixed>(
  'RCTMultilineTextInputView',
);
``` 

I can move "WindowsTextInput = requireNativeComponent('RCTTextInput');" from TextInput to the renamed *NativeComponent.js, but I'm not sure I know how to align the code with your suggestion here

Reply:
Ah, those components must not have switched to the codegen for the component. codegenNativeComponent is the new version of requireNativeComponent so yeah, you can just put requireNativeComponent in that file. 😀